Update Dockerfile

hos7ein opened the pull request:
Update Dockerfile

tidwall commented:
Hi @hos7ein, 

The current Dockerfile is designed to work in a specific CI environment. I'm concerned that merging your update will break that work flow.

Is there a specific problem with current Dockerfile that you changes that address?

hos7ein commented:
Hi @tidwall,

Honestly, current Dockerfile do not have problems, but I did some updates and options for users. For example:

- Updated alpine base image version.
- Updated Tile38 version. Also user can set specific version when they want.
- Downloaded Tile38 from GitHub releases page.
